Autopilot 1.3.0 docs (#914)
portworx/pxdocs: content/portworx-install-with-kubernetes/autopilot/how-to-use/operate-and-troubleshoot.md

This section provides common operational procedures for monitoring and troubleshooting your autopilot installation.

## Troubleshooting objects monitored by autopilot

### Get recent statuses using AutopilotRuleObjects

For each object monitored by Autopilot, it will create a corresponding `autopilotruleobject` instance in the namespace of the object.

> * For volumes (PVCs), it will be in the namespace of the PVC.

Reviewer: What is "it" here? the `autopilotruleobject`? the instance? I think we should use the noun and not the pronoun here for extra clarity.

Author: yeah it is autopilotruleobject. updated the PR.

* For storage pools, it will be in the namespace where Portworx is installed.

The `autopilotruleobject` is created only if an object's condiitons were atleast triggered once. In other words, you will not see an `autopilotruleobject` if the object was always in nornal state.

#### List all autopilotruleobjects 

> 

Reviewer: ```suggestion
The following command lists all Autopilot rule objects in all namespaces:
```

``` text
kubectl get autopilotruleobjects --all-namespaces
```

> You can also use the alias `aro`.

Reviewer: ```suggestion
Instead of entering the full `autopilotruleobjects` string, you can use the `aro` alias.
```

``` text
kubectl get aro --all-namespaces
```
```output
NAMESPACE   NAME                                       AGE
pg1         pvc-ec475444-e2d6-4533-863b-03263da7b04c   1s
```

#### Describe a specific object

> As seen below, the `Status` section will contain a list of recent statuses of the object.

Reviewer: ```suggestion
The `Status` section contains a list of recent object statuses:
```
